src/components/playground/PlaygroundComponent/PlaygroundComponent.jsx

```diff
@@ -23,6 +23,8 @@ export const PlaygroundComponent = () => {
   const request_input = useRef(null);
   const [request_info, setRequestInfo] = useState({});
   const [response_info, setResponseInfo] = useState({});
+  const [visibility_listener, setVisibilityListener] = useState(true);
+  const [scroll_direction, setScrollDirection] = useState("down");
```

Comment:
just a nitpick: do we use in this project double quotes?

Reply:
Haha, I understand. Well, in this case I prefer double quotes as this seems to be the standard within this project and thus would be more consistent.